pages/common/grep.md

```diff
@@ -21,7 +21,7 @@
 
 - Print 3 lines of context around, before, or after each match:
 
-`grep --{{context|before-context|after-context}} {{3}} "{{search_pattern}}" {{path/to/file}}`
+`grep {{--context|before-context|after-context}} {{3}} "{{search_pattern}}" {{path/to/file}}`
```

Comment:
```suggestion
`grep {{--context|--before-context|--after-context}} {{3}} "{{search_pattern}}" {{path/to/file}}`
```
 If we really want to change it, we should do it this way because of the interactive clients .